### PR TITLE
Merge pcap_ops.so into libtensorflow_io.so

### PR DESCRIPTION
**Pull request by yongtang**
to reduce the number of .so (and avoid potential duplicate copies of lib in multiple .so).
also expose tfio.IODataset.from_pcap().

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: tests/test_pcap_eager.py [hunk 1]
```diff
@@ -34,8 +31,19 @@ def test_pcap_input():
...
-  url_filenames = [file_url]
-  dataset = pcap_io.PcapDataset(url_filenames, batch=1)
+
+  dataset = tfio.IODataset.from_pcap(file_url, capacity=5).batch(1)
+
+  packets_total = 0
+  for v in dataset:
+    (packet_timestamp, packet_data) = v
+    if packets_total == 0:
+      assert packet_timestamp.numpy()[0] == 1084443427.311224 # we know this is the correct value in the test pcap file
```
Comment (BryanCutler) [inline tests/test_pcap_eager.py]: better to use `numpy.is_close`?

Comment (yongtang) [reply tests/test_pcap_eager.py]: @BryanCutler updated. Thanks.

File: tests/test_pcap_eager.py [hunk 2]
```diff
@@ -34,8 +32,19 @@ def test_pcap_input():
...
-  url_filenames = [file_url]
-  dataset = pcap_io.PcapDataset(url_filenames, batch=1)
+
+  dataset = tfio.IODataset.from_pcap(file_url, capacity=5).batch(1)
+
+  packets_total = 0
+  for v in dataset:
+    (packet_timestamp, packet_data) = v
+    if packets_total == 0:
+      assert np.isclose(packet_timestamp.numpy()[0], 1084443427.311224) # we know this is the correct value in the test pcap file
```
Comment (BryanCutler) [inline tests/test_pcap_eager.py]: since it's a big number you might want to add the `rtol` param to something like `rtol=1e-15`

Comment (yongtang) [reply tests/test_pcap_eager.py]: @BryanCutler Updated. Thanks.

**Pull request merged**